src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { LinkArrowIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  label: string;
+  href: string;
+  size?: 'small' | 'medium' | 'large';
+  outlined?: boolean;
+  rounded?: boolean;
+  arrow?: boolean;
+  arrowSize?: number;
+  arrowColor?: 'black' | 'white';
+  textRegular?: boolean;
+  className?: string;
+};
+
+export const LinkCustom = ({
+  label,
+  href,
+  size = 'large',
+  outlined = false,
+  rounded = false,
+  arrow = true,
+  arrowSize = 24,
+  arrowColor = 'white',
+  textRegular = false,
+  className = '',
+  ...props
+}: LinkCustomProps) => {
+  const btnClass = classNames(styles.button, styles[size], `stroke:${arrowColor}`, {
+    [styles.outlined]: outlined,
+    [styles.colored]: !outlined,
+    [styles.text]: !arrow,
```

Comment:
As I can see it's only `margin-top` defined here. Do we really need it?
![Screenshot 2024-05-13 at 13 03 23](https://github.com/rolling-scopes/site/assets/56927722/714e8ad1-31ca-4781-b4da-a7cb1f93845f)

Reply:
Removed